### ISSUE TITLE
Replace pagination code from utility function paginated_queryset in participants app

### ISSUE THREAD

**Issue by TroJan** [Cloud-CV/EvalAI#444]
`paginated_queryset` is a function in `base utils.py`.

Idea is to remove paginator code with a function named `paginated_queryset`

```
paginator = PageNumberPagination()
paginator.page_size = settings.REST_FRAMEWORK['PAGE_SIZE']
result_page = paginator.paginate_queryset(participant_teams, request)
```

For reference, look at PR #422

**Comment by anweshknayak** [comment]
can I work on this ?

**Comment by deshraj** [comment]
@anweshknayak I think you are already working on #300. Please try to finish that first.

**Comment by parismita** [comment]
shall i try this??

**Comment by deshraj** [comment]
Sure. Assigning it to you.

**Comment by parismita** [comment]
@deshraj please check

**Comment by deshraj** [comment]
Great work @parismita. :-) 

Fixed in #447. Closing it.
